syntax = "proto3";

message User {
    string userName = 1;
    string password = 2;
}

message Account {
    int accountNum = 1;
    string ssn = 2;
    string firstName = 3;
    string lastName = 4;
    string openDate = 5;
}

// singular object
message Bank {
    repeated Account allAccounts = 1;
    repeated User allUsers = 2;
}